docs/readiness/adr-001-interchain-accounts.md

```diff
@@ -0,0 +1,82 @@
+---
+ADR: 001
+Title: Interchain Accounts
+Status: Draft Implements
+Category: Feature
+Author: Sean King & Damian Nolan
+Created: 2022-01-19
+Mdified: 2022-01-19
+Requires: Cosmos-SDK, go-ibc
+Required-By: mauth
+Implements: Interchain Accounts
+---
+
+# ADR 001: Interchain
+
+## Changelog
+
+- 2022-02-04: added content
+- 2022-01-19: init
+
+## Abstract
+
+This is the Core Interchain Accounts Module. It allows the Cosmos Hub to act as a host chain with interchain accounts that are controlled by external IBC connected "Controller" blockchains. Candidate chains include Umee, Quicksilver, Sommelier. It is also a necessary component for a Authentication Module that allows the Cosmos Hub to act as a Controller chain as well. This will be recorded in a separate ADR.
+
+## Rationale
+This allows the Hub to participate in advanced cross-chain defi operations, like Liquid Staking and various protocol controlled value applications.
+
+## Desired Outcome
+The hub can be used trustlessly as a host chain in the configuration of Interchain Accounts.
+
+## Consequences
+The hub will be the target for protocol controlled accounts, it is not expected to have the same risks as atomic smart contract controlled accounts which added re-entrancy bugs as a security vulnerability on EVM chains, but it does provide a similar capability.
+
+#### Backwards Compatibility
+This is the first of its kind.
+
+#### Forward Compatibility
+There are future releases of Interchain Accounts which are expected to be backwards compatible.
+
+## Technical Specification
+[ICS-27 Spec](https://github.com/cosmos/ibc/blob/master/spec/app/ics-027-interchain-accounts/README.md)
+
+## Development
+- Integration requirements
+  - Development has occured in [IBC-go](https://github.com/cosmos/ibc-go) and progress tracked on the [project board](https://github.com/orgs/cosmos/projects/7/views/7) there.
+- Testing (Simulations, Core Team Testing, Partner Testing)
+  - Simulations and Core Team tested this module
+- Audits (Internal Dev review, Third-party review, Bug Bounty)
+  - An internal audit, an audit from Informal Systems, and an audit from Trail of Bits all took place with fixes made to all findings.
```

Comment:
Where do these live, might be worth linking?

Reply:
not yet publicly available